libc/test/src/stdio/sprintf_test.cpp

```diff
@@ -3201,6 +3201,166 @@ TEST_F(LlvmLibcSPrintfTest, FloatAutoLongDoubleConv) {
 
 #endif // LIBC_COPT_PRINTF_DISABLE_FLOAT
 
+#if defined(LIBC_COMPILER_HAS_FIXED_POINT) &&                                  \
+    !defined(LIBC_COPT_PRINTF_DISABLE_FIXED_POINT)
+TEST_F(LlvmLibcSPrintfTest, FixedConv) {
+
+  // These numeric tests are potentially a little weak, but the fuzz test is
+  // more thorough than my handwritten tests tend to be.
+
+  // TODO: Commit the fuzzer
```

Comment:
Stale comment as of https://github.com/llvm/llvm-project/pull/82707/commits/553c7d9a267465d0eb3bf068454db62e0af01c97